server/cache.go

```diff
@@ -408,7 +408,7 @@ func newClusterInfo(clusterRoot string) *ClusterInfo {
 	return cluster
 }
 
-func (c *ClusterInfo) addStore(store *metapb.Store) {
+func (c *ClusterInfo) addMetaStore(store *metapb.Store) {
```

Comment:
Meta store is only for `metapb.Store`, so i use meta for this type of store, maybe both can be merged to one function.